### ISSUE TITLE
Move dynamorio.org to Github Pages

### ISSUE THREAD

**Issue by derekbruening** [DynamoRIO/dynamorio#2860]
Today dynamorio.org is hosted on bluehost with completely separate content and edit authorization from the Github project.  We should move it to Github Pages which will allow project members to edit the pages and will put everything in a centralized location.  CDash may need to remain on bluehost but the other content could all be moved pretty easily, including the Doxygen-generated html pages.

**Comment by derekbruening** [comment]
Once this is done, we should have our cron builds (#1967) publish the html docs to the live web site.

**Comment by AssadHashmi** [comment]
Is there anything I can do to help with this issue? Is it a case of moving the whole of dynamorio.org to GitHub in one go or to do it incrementally? My interest is primarily in moving the Doxygen html pages and adding generation to the cron build, but I don't mind doing the whole lot if it doesn't take too much extra time.

**Comment by derekbruening** [comment]
I split off publishing the cronbuild docs as #3194.  Happy to let someone else set it up :)  Once there is a Github Pages framework it should be easy to move the other content.

**Comment by derekbruening** [comment]
I'm going to try to do this this weekend.

First decision: one web site for whole DynamoRIO org with drmemory or other project subdirs or whatnot, or separate web site per repo (i.e., drmemory and dynamorio separate)?

I think there are few advantages to per-repo web sites: there will be shared configs and content and it will be simpler to have a single web site.  So I'm going to make a separate repo for the single web site.  drmemory.org can either be parked on a subdir, or just an alias and resolve to dynamorio.org/drmemory.

**Comment by derekbruening** [comment]
This is now complete: the website contents are in the `dynamorio.github.io` repository.  I am adding instructions for how to test local changes and contribute them to the README.md there.
